test/muxer_test.exs

```diff
@@ -109,7 +110,8 @@ defmodule Membrane.Matroska.MuxerTest do
         stream_format: %Opus{channels: 2, self_delimiting?: false}
       })
       |> via_in(Pad.ref(:input, @pad_id_4))
-      |> get_child(:muxer)
+      |> get_child(:muxer),
```

Comment:
Why?

Reply:
This way it was simpler for me to compare it with versions from core v0.10.

Reply:
Ok, then